### ISSUE TITLE
Dragon tutorial

### ISSUE THREAD

**Issue by AkyV** [TokyoSU/Plugin_TokyoSU#41]
The dragon setup is too complicated. We need a tutorial.

**Comment by TokyoSU** [comment]
it's not really difficult but i will create a tutorial later, you need:
- TR2_DRAGON_FRONT/BACK
- TR2_DRAGON_EXPLOSION1/2/3
- TR2_DRAGON_BONE_FRONT/BACK
- TR2_MARCO_BARTOLI
- TR2_DRAGON_FIRE
now after all customize is setup and object in the wad/level.
place marco_bartoli in the level with a big room and trigger it :)

**Comment by AkyV** [comment]
You are right. I was able to make it work easily this way. But what I noticed:
- Bartoli's trigger should be One Shot, or else the trigger won't work till Lara is in the trigger zone. (Documentation.)
- The flashing at the spawning of the dragon stops if he starts walk. It restarts when he falls down - but don't stop again if he stand up again. It ends only at his death.
- Documentation: for Enemy customization, use Dragon Front.
- Sometimes it is really hard, almost impossible to grab the knife.
- If I add the knife-puller animation from Xian.tr2 to my WAD, how can I call it?
- Xian.tr2 also has a special Lara death animation. It should be typed at Dragon Front as an OCB?
- Documentation (or bug): saving the game while the flesh is just comes down off the dead dragon makes her alive again after the load.
- Bones aren't saved.

**Comment by TokyoSU** [comment]
1) i used the simple trigger and it work for me :x 
2) it's normal since it's already spawned.
4) collision code from TR2, maybe a custom one is required.
5) it's hardcoded
6) it's not from the dragon i think since i dont view any code for that.
7) yea save bug again, this save shit is anoying, how core design have coded that seriously, the save flags is set :(

**Comment by AkyV** [comment]
1. It also works for me, but a One Shot seems recommended.
2. But why it does not work at his first spawned state, if it works at his further respawned state?
4. Hardcoded, but as a LARA object animation? And with which AnimID? (In Xian.tr2 it is NOT a LARA object anim.)
